src/test/scala/no/ndla/articleapi/controller/ArticleControllerV2Test.scala

```diff
@@ -150,4 +152,18 @@ class ArticleControllerV2Test extends UnitSuite with TestEnvironment with Scalat
       convertedBody should equal(allLicenses)
     }
   }
+
+  test("GET / should use size of id-list as page-size if defined") {
+    val searchMock = mock[SearchResult]
+    val searchResultMock = mock[io.searchbox.core.SearchResult]
+    when(articleSearchService.all(any[List[Long]], any[String], any[Option[String]], any[Int], any[Int], any[Sort.Value], any[Seq[String]]))
+      .thenReturn(searchMock)
+    when(searchMock.response).thenReturn(searchResultMock)
+    when(converterService.getHitsV2(searchResultMock, "nb")).thenReturn(Seq.empty)
+
+    get("/test/", "ids" -> "1,2,3,4", "page-size" -> "10", "language" -> "nb") {
```

Comment:
Må du ikke ha `page-size < # of ids` for at denne testen skal teste det du vil?

Reply:
Ah. Jo du har rett. Jeg var litt for kjapp. (Og ble fintet ut av at jeg tenkte normaltilfellet var å ha flere id'er enn pagesize)...